Misc/NEWS.d/next/Library/2020-01-21-16-38-25.bpo-39411.9uHFqT.rst

```diff
@@ -0,0 +1,3 @@
+Rewrite :mod:`pyclbr` with the support of abstract syntax trees. Add an
```

Comment:
The rewrite is not a user-facing change so we should not mention it here. The part to `is_async` on the other hand is important.